lightspeed/modules/proc_create-oauth-app.adoc

```diff
@@ -0,0 +1,39 @@
+:_content-type: PROCEDURE
+
+[id="create-oauth-app_{context}"]
+
+= Creating an OAuth application
+
+Use this procedure to create an OAuth application for your {LightspeedShortName} on-premise deployment. 
+
+.Prerequisites
+* You have an operational Ansible automation controller instance.
+
+.Procedure
+. Log in to the automation controller as an administrator.
+. Under *Administration*, click *Applications > Add*. 
+. Enter the following information:
+.. *Name*: Specify a unique name for your application.
+.. *Organization*: Select a preferred organization.
+.. *Authorization grant type*: Select *Authorization code*.
+.. *Redirect URIs*: Enter a temporary URL for now, for example, `https://temp/`.
++
+The exact {LightspeedShortName} application URL is generated after the on-premise deployment is completed. After the deployment is completed, you must change the Redirect URI to point to the generated {LightspeedShortName} application URL. For more information, see xref:update-redirect-uri_configuring-lightspeed-onpremise[Updating the Redirect URIs].
+
+.. From the *Client type* list, select *Confidential*.
+
+. Click *Save*. 
```

Comment:
You could use the asciidoc button macro in this line (and throughout) for consistency. Info about the button macro here: https://docs.asciidoctor.org/asciidoc/latest/macros/ui-macros/

Reply:
@hherbly : I noticed there are multiple instances throughout the docs, for different features, that need to be updated. I've created a separate JIRA https://issues.redhat.com/browse/AAP-25608 for this clean-up activity.